digitalutsc/islandora_solution_pack_oralhistories
# remove docs, moved to wiki

# What does this Pull Request do?
Removes the docs directory, docs now maintained in wiki: https://github.com/digitalutsc/islandora_solution_pack_oralhistories/wiki

# What's new?
removed docs!

# How should this be tested?
Make sure docs isn't in the repo, make sure all docs + images moved to wiki: https://github.com/digitalutsc/islandora_solution_pack_oralhistories/wiki

# Additional Notes:
# Interested parties
